src/main/java/com/iota/iri/service/API.java

```diff
@@ -120,6 +120,8 @@
 
     private final String[] features;
 
+    private static Object lock = new Object();
```

Comment:
I am afraid the naming "lock" is misleading.

Reply:
I was thinking that it should be acting like a lock.
I want to know about you concern more that I can come up with a better naming.

Reply:
There might be more than 2 locks within this class.

Reply:
I change it from `lock` to `PoWTimeLock`.